code/modules/research/techweb/_techweb.dm

```diff
@@ -48,6 +50,18 @@
 	id = "SCIENCE"
 	organization = "Nanotrasen"
 
+/datum/techweb/bepis	//Should contain only 1 BEPIS tech selected at random.
+	id = "SCIENCE"

```

Comment:
I'm not sure id's are used for anything but it probably shouldn't be the same as default research web.